### ISSUE TITLE
[CAIA a11y Research - PILOT]: Benefits Decision Reviews, Supplemental Claims - Pilot ONLY (September 2024)

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#92043]
> [!IMPORTANT]
> - **TICKET STATUS:** `COMPLETED` ✅ 
> - **PILOT DATE:** 9/12/2024 at 3:30 p.m. 
> - **TYPES OF ASSISTIVE TECH:** Mobile or Desktop Works `Desktop - Windows/NVDA`
> - **PROTOTYPE:** [Prototype in Staging](https://staging.va.gov/decision-reviews/test-sc/introduction)
> - **PILOT PREP NOTES:** [Link](https://docs.google.com/document/d/1GFqOnmfAjoxUr7qhCPXeAMX1KniIpAKgxILbJ3Gvljc/edit)

# CAIA A11y Support
This is the **only ticket** for this research - created to track work, completed from this pilot session.

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @mutiaadhocteam   (pilot)
- @coforma-jamie   (co-pilot seat 1)
- @sara-amanda (co-pilot seat 2)

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Benefits Decision Reviews
- OCTO product owner: @amylai-va
- Product manager:
- Designer: Eileen Chi
- FE engineer: Robin Garrison
- Product/team Slack channel: #benefits-decision-reviews

</details>

## Description

> We only need a pilot participant for one session between 9/9-9/13

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** 9/12/2024 at 3:30 p.m.
-  **Prototype: [Staging](https://staging.va.gov/decision-reviews/test-sc/introduction)
- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

#### Do you have an accessibility champion?

Yes, we have a champion

#### If you have an accessibility champion, what's their name?

David Kennedy

</details>

## Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @NaomiPMC in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/decision-reviews/Supplemental-Claims)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1720452215264/52063e8eae03808ccd0facce0539f52f332ad20e?sender=ud193ce75f65a17a316ea6194)
- [Prototype or mockups](https://staging.va.gov/decision-reviews/test-sc/introduction)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Supplemental-Claims/Research/0924-2024-SC-Updates/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Supplemental-Claims/Research/0924-2024-SC-Updates/conversation-guide.md)
- [Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82219)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y**
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**
- [x] **Update CAIA a11y Capacity Calendar:** `Holds placed for 9/12 and 9/13`
- [x] **Update CAIA a11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] Research Tracker Updated (Placeholder Info)
- [x] Research Tracker Updated (Scheduled Info)
- [ ] Research Tracker Updated (Post-Research Info)
- [x] **Review research** conversation guide
- [x] Request AT device and tech to be used in sessions and preference for testing in pilot session
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot - _**CAIA and the team will determine what AT will be used in this session, specifically.**_
- [x] **Review prototype** in `Staging`
- [x] **Run pilot** with VFS Researcher
- [x] **Optional: [Note Taking](https://docs.google.com/document/d/1GFqOnmfAjoxUr7qhCPXeAMX1KniIpAKgxILbJ3Gvljc/edit):** If there is only one a11y, notes cannot be taken due to pilot participation.
- [x] [Additional Resource for Voice Control Testing](https://accessibility-manual.dwp.gov.uk/best-practice/voice-controller-testing)
```

**Comment by sara-amanda** [comment]
Hi, @eileen-coforma ! Happy to schedule that pilot for you this week! What AT and device are you looking to test, specifically?

**Comment by coforma-jamie** [comment]
@eileen-coforma, took a look at the script and have a couple of thoughts/questions:
- Does the participant need to log in to access the prototype? For AT users in particular, this can take a bit, and should be accounted for in the script.
- Make sure to ask the participant if they're using AT, and what AT they're using. Some folks end up joining sessions using different device + AT pairs than they signed up with (one example: it's common to see folks sign up PC + JAWS, but use iPhone + VoiceOver during the session). Ask early, during the "Intro" section - if they aren't using the tools you're looking for (or aren't AT users at all), you can end the session early.
- A general note about hearing screen reader audio:
   - On desktop, when the participant is sharing their screen, ask them to select the "Share audio" checkbox. You'll be able to hear their SR audio clearly.
   - This unfortunately doesn't work on mobile. There's no "Share audio" option, and you likely won't be able to hear the SR audio at all (or just barely). You'll be able to visually see a thick focus outline wherever the SR is focused, and you can ask the participant to talk through what they're doing.

**Comment by eileen-coforma** [comment]
@coforma-jamie 

Thank you for taking a look at the script.

> Does the participant need to log in to access the prototype? For AT users in particular, this can take a bit, and should be accounted for in the script.

No, we purposely coded the prototype to have a workaround for the login, and the information will all be mock info

> Make sure to ask the participant if they're using AT, and what AT they're using. Some folks end up joining sessions using different device + AT pairs than they signed up with (one example: it's common to see folks sign up PC + JAWS, but use iPhone + VoiceOver during the session). Ask early, during the "Intro" section - if they aren't using the tools you're looking for (or aren't AT users at all), you can end the session early.

Yes, we will ask about the device they plan to use during the warm-up as a confirmation (since we get a list of what they would be using ahead of time). Sometimes we move tech setup to the beginning of the conversation guide for AT-sessions because it's easier to get the technology bit out of the way before diving in

**Comment by sara-amanda** [comment]
> [!CAUTION] 
> **This pilot feedback is a `WIP` and not yet complete.** 
> _Please check back, when tags are added to notify the team, and this cautionary message has been removed. Thanks so much!_ 

## Pilot Feedback from Session

**Feedback specific to:** Screen Readers (NVDA)

### Research Goals & the Pilot 

Research Goal Status | Met | Not Met | Partially Met
---- | ---- | ---- | ----
Determine if the Supplemental Claims updates create any new points of friction within the form |   |   | X
Validate if Veterans understand (or skip if ineligible) the series of homelessness questions | X |   |  
Validate that users are able to understand if they’re eligible for and respond to the VHA MST Notifications question | X |   |  
Validate that users are able to understand what it means to submit a piece of non-VA evidence (submitting a 4142/a) |   |   | X
Validate that the users are able to complete the new Supplemental Claim flow | X |   |  
Validate that AT-users are able to navigate through the entire form | X |   |  

### Pilot Takeaways 

#### Form Structure
- Content could be simplified and cut down to reduce cognitive load. 
    - **DK**: Javascript managing focus, may not trigger fast enough, if the user is skimming a lot. 
- Form was lengthy
- Multiple fields makes the form more complicated. 

#### Bugs
- Existing Bug: Some buttons are not announcing their actions. (Edit Button)
    - It announces what you will be editing. 
    - Once clicked it does not announce the action. 
- Existing Bug: Step 5 of 5: Clicking a button took the user back to the top of the page. 
- Potential Bug: Clicking the additional component button. 
    - Text appeared on one. 
    - Text did not appear on another. 

#### Additional Behavior Observed
- Notification did not alert when it is closed. 
- Some additional info. components were not expanding for the user.
- We recommend your team double checks how the error reads out - in this section: “Add a first VA or military treatment location” 

### Quotable Moments

> "I like the error messages and error handling of the form."

> “I like the fact, like before with the edit buttons, it announces what you will be editing.” 

> “Hit an edit button and it has not announced anything.” 

> “I like that it tells me what I am updating as well. It also shot me back to where I was before, which was the edit button.” 

> “It seems a bit wordy, all of the information, if they could make it more concise, so I am not narrowing down each paragraph.” 

> “I like the way you can go through and collapse and expand as desired because there are so many sections.” 

> "I just got pushed all over the place by the focus, not sure why" (user was on step 4 of 5 and it was a required question.


## Overall Confidence Level 
`6` out of `10` - per pilot user feedback 

### Rationale  
- There were a few parts that contributed to a bit of confusion. 
- Confusion was related to mostly items a sighted user would be able to see and imply, making it "patchy for this user." 
- The length, multiple fields and wording, also added to the rating.

**Comment by mutiaadhocteam** [comment]
**Extra Feedback**

- Controls that must be expanded for users to access pertinent information about the process should be reduced. Users may consider these controls to be cumbersome and disruptive to the experience. Additionally, if a user is skimming the pages, the controls may not be caught by the user.
- Pages containing only one question or a limited number of radio buttons, form fields, etc. were ideal! This prevents the user from confusion in regard to their location on the page.
- There was a bug when activating the checkbox required to submit the form on step 5. Upon activating the checkbox, focus was sent back to the main heading of the page. Users may experience confusion when this occurs.
